### DISCUSSION TITLE
Security Issue with the Rustdesk Client in a remotesession

### DISCUSSION THREAD

**Discussion by ukdt7** [rustdesk/rustdesk#5784] [General]
### Bug Description

If you are connected to a remotesession and you open the rustdesk client from your partner. the windows is greyed out, if you put your cursor onto the windows. that doesnt count to the titlebar of the window. if you click on it an use your keyboard, you can easily control the remote rustdesk client and see settings and connect to other clients. i'm not sure how bad this is, but it could be a vulnerability with potential.

### How to Reproduce

just connect to your partner and open his rustdesk client. click onto the titlebar and now use your keyboard to control your partners rustdesk client.

### Expected Behavior

a full lock of the application OR a transformation of the UI to the controlpanel of the established connection. 

maybe you can transform the client to the controlpanel of the connection so you cant configure the client while someone is connected to. i think that would be better.

### Operating system(s) on local side and remote side

Win10 22h2

### RustDesk Version(s) on local side and remote side

1.2.2

### Screenshots

i can send you screenshots, if you have questions

**Comment by rustdesk** [comment]
We will remove the "greyed out"  since there is already "Unlock security settings" button.
